modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/resource/v2_0/PriceEntryResourceImpl.java

```diff
@@ -272,13 +272,6 @@ else if (Validator.isNotNull(skuExternalReferenceCode)) {
 		DateConfig expirationDateConfig = DateConfig.toExpirationDateConfig(
 			priceEntry.getExpirationDate(), serviceContext.getTimeZone());
 
-		boolean priceOnApplication = false;
-
-		if (commercePriceList.isCatalogBasePriceList()) {
-			priceOnApplication = GetterUtil.getBoolean(
-				priceEntry.getPriceOnApplication());
-		}
-
```

Comment:
I don't get why? I think this should be done with the https://issues.liferay.com/browse/COMMERCE-11276